app/views/track_stream/index.erb

```diff
@@ -20,4 +20,7 @@
       }
     %>
   </div>
+
+  <%= erb :"gitter/chat_button", locals: { control: stream } %>
```

Comment:
I think we can fetch `track` from `TrackStream` and `Submission` without problems since `TrackStream` has a [track_id](https://github.com/exercism/exercism.io/blob/10591be6d5de31a789fab0e5e0cebe15716b766c/lib/exercism/track_stream.rb#L7) and `Submission` has a [track](https://github.com/exercism/exercism.io/blob/10591be6d5de31a789fab0e5e0cebe15716b766c/lib/exercism/submission.rb#L92) already. Another thing that I didn't notice is that `track_id` is the same as `language`? At least for the `Submission` class.